crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -788,6 +788,18 @@ impl<'w, 's> Commands<'w, 's> {
     ) -> EntityCommands {
         self.spawn(Observer::new(observer))
     }
+
+    /// Sends an arbitrary [`Event`].
+    ///
+    /// This is a convenience method for sending events without requiring an [`EventWriter`]. The event is sent
+    /// via exclusive world access when commands are applied. Note that for performance critical or very frequently
+    /// sent events it will be faster to use a typed [`EventWriter`] instead.
```

Comment:
```suggestion
    /// This is a convenience method for sending events without requiring an [`EventWriter`]. 
    /// ## Performance
    /// Since this is a command, exclusive world access is used, which means that it will not profit from
    /// system-level parallelism on supported platforms.
    /// If these events are performance-critical or very frequently
    /// sent, consider using a typed [`EventWriter`] instead.
```